modules/ROOT/pages/release-notes/release-notes55.adoc

```diff
@@ -50,25 +51,25 @@ For information on:
 
 For information on:
 
-* The `lineheight` toolbar button, see: link:{baseurl}/advanced/available-toolbar-buttons/#thecoretoolbarbuttons[Available Toolbar Buttons - The core toolbar buttons].
-* The `lineheight` menu item, see: link:{baseurl}/advanced/available-menu-items/#the-core-menu-items[Available Menu Items - The core menu items].
-* The `LineHeight` editor command, see: link:{baseurl}/advanced/editor-command-identifiers/#coreeditorcommands[Commands available for {productname} - Core Editor commands].
-* The `LineHeight` query command value, see: link:{baseurl}/advanced/editor-command-identifiers/#availablequerycommandvalues[Commands available for {productname} - Available query command values].
+* The `lineheight` toolbar button, see: xref:advanced/available-toolbar-buttons.adoc#thecoretoolbarbuttons[Available Toolbar Buttons - The core toolbar buttons].
+* The `lineheight` menu item, see: xref:advanced/available-menu-items.adoc#thecoremenuitems[Available Menu Items - The core menu items].
```

Comment:
This xref doesn't resolve as the `thecoremenuitems` id doesn't exist on the relevant heading in `available-menu-items.adoc`. e.g:
```
[[thecoremenuitems]]
== The core menu items
```

Reply:
done